src/targetNumber.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const targetNumber = Math.round(Math.random() * 9000 + 999).toString();
+
```

Comment:
This will generate duplicate digits:
![image](https://github.com/mate-academy/node_bulls-and-cows/assets/23013525/96e31a03-e52f-4a34-8833-7835b939073d)